engine/src/main/java/org/destinationsol/game/SolGame.java

```diff
@@ -107,6 +107,9 @@
 
     public SolGame(String shipName, boolean isTutorial, boolean isNewGame, CommonDrawer commonDrawer, Context context,
                    WorldConfig worldConfig) {
+        // TODO: make this non-static
+        FactionInfo staticIgnore = new FactionInfo();
```

Comment:
Couldn't you, like, at least make a static `init()` method to call instead? This is ugly.